tests/gdimagefill/bug00104_1.c

```diff
@@ -0,0 +1,40 @@
+/* $Id$ */
+#include "gd.h"
+#include <stdio.h>
+#include "gdtest.h"
+
+int main()
```

Comment:
It is a similar one, but instead of writing the image to an output file, I am using gdAssertImageEqualsToFile like the rest of the tests. I also used different size image, but it's the same bug, as the bug occurs in any image with width > height. I changed the variable names to adhere to the same names used in the other tests (I didn't know if there is a convention, to I used the one in the other files).

So basically it's the same text.